samples/ControlCatalog/Pages/PopupsPage.xaml

```diff
@@ -0,0 +1,18 @@
+<UserControl x:Class="ControlCatalog.Pages.PopupsPage"
+             xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  
+  <StackPanel Orientation="Vertical" Spacing="10">
+    
+    <TextBlock Text="'Light Dismiss' Popup" />
+    <Button Content="Show Popup" Click="ButtonLightDismiss_OnClick" />
+
+    <TextBlock Text="Popup that stays open" />
+    <Button Content="Show Popup" Click="ButtonPopupStaysOpen_OnClick" />
+
+    <TextBlock Text="TopMost popup that stays open" />
+    <Button Content="Show Popup" Click="ButtonTopMostPopupStaysOpen" />
```

Comment:
These testing buttons can be moved to the IntegrationTestApp. I can try to add integration test on top of them later.

Reply:
I could use some guidance on this one. Not sure what I should do 😳